test/cpp/api/modules.cpp

```diff
@@ -4865,6 +4884,12 @@ TEST_F(ModulesTest, PrettyPrinCrossEntropyLoss) {
       c10::str(CrossEntropyLoss()), "torch::nn::CrossEntropyLoss()");
 }
 
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+TEST_F(ModulesTest, PrettyPrinCrossEntropyLossWithSoftLabels) {
```

Comment:
nit: PrettyPrin -> PrettyPrint

Reply:
Fixed for the original CrossEntropy as well